src/metabase/api/dashboard.clj

```diff
@@ -590,7 +590,10 @@
         _                                       (when (seq to-update)
                                                   (update-tabs! dashboard to-update))]
     {:temp->real-tab-ids temp-tab-id->tab-id
-     :deleted-tab-ids    to-delete-ids}))
+     :deleted-tab-ids    to-delete-ids
+     :num-tabs-created   (count to-create)
+     :num-tabs-deleted   (count to-delete)
+     :total-num-tabs     (+ (count to-create) (count to-update))}))
```

Comment:
The total number of tabs is `(count new-tabs)`. It can be calculated in `create-dashcards!` and doesn't need to be returned by `do-update-tabs!.

Reply:
yes we could get it from there but I prefer doing this because:
- having all tabs stats related come from `do-update-tabs!`
- the number is returned after doing the operation, thus being a bit more reliable?